assets/js/googlesitekit/datastore/modules/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * core/modules data store
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { createNotificationsStore } from '../../data/create-notifications-store';
+import modules from './modules';
+
+const notifications = createNotificationsStore( 'core', 'modules', 'notifications' );
```

Comment:
Not sure why you added a notifications part to the store here? There are no general notifications for modules, also this wasn't part of the ACs :)

Reply:
Yeah, I saw that, but I assumed we might want notifications for them, but I guess they'd be scoped to the modules themselves? I figured… maybe registering a module failed, etc. But I'll remove it for now—we can add it later if needed.